code/game/objects/items/holy_weapons.dm

```diff
@@ -761,3 +761,138 @@
 	attack_verb_continuous = list("stabs", "pokes", "slashes", "clocks")
 	attack_verb_simple = list("stab", "poke", "slash", "clock")
 	hitsound = 'sound/weapons/bladeslice.ogg'
+
+/obj/item/clothing/head/helmet/chaplain/knight1
+	name = "holy knight helm"
+	desc = "A great helm with a gryphon on top of it. Looks fancy!"
```

Comment:
damn bro I didn't know that these helmets would come with real live griffons on top of them